lib/elastix/mapping.ex

```diff
@@ -45,29 +45,18 @@ defmodule Elastix.Mapping do
   @spec get(
           elastic_url :: String.t(),
           index_names :: String.t() | list,
-          type_names :: String.t() | list,
           query_params :: Keyword.t()
         ) :: HTTP.resp()
-  def get(elastic_url, index_names, type_names, query_params \\ [])
+  def get(elastic_url, index_names, query_params \\ [])
 
-  def get(elastic_url, index_names, type_names, query_params)
-      when is_list(type_names) and is_list(index_names) do
-    prepare_url(elastic_url, make_path(index_names, type_names, query_params))
```

Comment:
I guess all these aren't needed anymore?

Reply:
as part of elasticsearch bumping past 7.0 (I think?), type names are not needed for sending in the request, as, from what I've seem, the endpoint is completely based on the index